src/common/logger/tests/logger_test.cpp

```diff
@@ -3,7 +3,7 @@
 #include <gtest/gtest.h>
 #include <spdlog/sinks/ostream_sink.h>
 #ifdef __APPLE__
```

Comment:
we might not need to differentiate between `__APPLE__` and `__linux__`, since they rely on the same implementation. Have you tried including only `<spdlog/sinks/stdout_color_sinks.h>` and removing the other includes below? Specifically:
```
#include <spdlog/sinks/syslog_sink.h>
#include <spdlog/sinks/systemd_sink.h>
#include <spdlog/sinks/stdout_color_sinks.h>
```

Reply:
Done. Redirected logger to stderr while at it. stderr sink also lives in `stdout_color_sinks.h` which is convenient.